src/scripts/exclusion_term_expansion.py

```diff
@@ -297,13 +297,19 @@ def run(
         df_results = pd.DataFrame()
         df_results['term_id'] = ''
 
+    # Filter out non-owned
+    owned_prefixes = list(onto_config['base_prefix_map'].keys())
+    df_results['prefix'] = df_results['term_id'].apply(lambda x: x.split(':')[0])
+    df_results = df_results[df_results['prefix'].isin(owned_prefixes)]
+    df_results.drop('prefix', inplace=True, axis=1)
+
     # Sort
     if len(df_results) > 0:
         df_results = df_results.sort_values(['term_id', 'exclusion_reason'])
 
     # Add simplified version
     # - df_results_simple: Simpler dataframe which is easier to use downstream for other purposes
-    df_results_simple = df_results[['term_id']]
+    df_results_simple = pd.DataFrame(set(df_results['term_id']))
```

Comment:
FYI: there were some duplicate terms in the output, because some of the terms got excluded for multiple reasons. This will still show up in the robot TSV output, because the exclusion reason is one of the columns. But for the TXT output, it's just a flat list of terms, so I removed any duplicates.

Reply:
Interesting, and weird. Please make an issue about it with some examples and ask Sabrina to take a look.

Reply:
Done! https://github.com/monarch-initiative/mondo-ingest/issues/128